### PR TITLE
Feat contact page

### PR DESCRIPTION
**Pull request by BastinJafari**
https://github.com/jina-ai/website/issues/41

### CODE REVIEW

File: src/pages/contact.tsx [hunk 1]
```diff
@@ -1,17 +1,125 @@
...
+                />
+                <ContactCard
+                  title={websiteCopyStrings.contactInfoCard4Header}
+                  infoList={websiteCopyStrings.contactInfoCard4InfoList}
+                />
+              </div>
+            </div>
+
+            <div className="flex-1">
+              <h2 className="flex flex-row items-center justify-center text-2xl font-bold mb-10">
```
Comment (Roshanjossey) [inline src/pages/contact.tsx:112]: I think this section needs to be linked to our slack invite link

File: src/utils/websiteCopyStrings.ts [hunk 2]
```diff
@@ -123,5 +123,31 @@ export const websiteCopyStrings = {
...
     'Our Flow architecture follows a modular design. Easily plug and play with existing solutions!',
   contributeResourcesNewsletterLearnMoreLink: 'https://github.com/jina-ai/jina',
-  contributeResourcesLearnMore: 'Learn more'
+  contributeResourcesLearnMore: 'Learn more',
+  contactInfoHeaderPart1: 'Contact',
+  contactInfoHeaderPart2: 'info',
+  contactInfoCard1Header: 'STAY IN TOUCH',
+  contactInfoCard1InfoList: ['711-2880 Nulla St.', 'Mankato Mississippi 96522'],
+  contactInfoCard2Header: 'CONTACT',
+  contactInfoCard2InfoList: [
```
Comment (Roshanjossey) [inline src/utils/websiteCopyStrings.ts:132]: The phone number and email is wrong.

**Comment by Roshanjossey** [comment]
What happens when user submits their contact information? Do we want to do that separately?

**Comment by BastinJafari** [comment]
There is a research ticket about that

**Pull request merged**